### ISSUE TITLE
additional view or filter needed

### ISSUE THREAD

**Issue by insilentio** [thomashartmeier/dorfbraeu-admin#14]
The view "offene Bestellungen" is too wide for accounting purposes. I would need solely a view of the orders with a need to send an invoice ("Rechnung noch nicht geschickt"), otherwise it gets way too cumbersome to keep the overview. (instead of a separate view, I could also live with filtering options, but the view is preferrable)

Additionally, I would suggest to remove the status "Bestellung bezahlt". It makes no sense to maintain this here because the accounts payable are maintained in GnuCash. Maintaining here would just cause additional work with no benefit at all.

**Comment by thomashartmeier** [comment]
Good point! I can add a new view for accounting purposes 👍🏼  Not anymore tonight, though... 😉 

The second point is done with my latest two commits, c788c2b5 and c2fe9010.
